docusign-connector/src/com/axonivy/connector/docusign/auth/JwtFactory.java

```diff
@@ -99,7 +99,7 @@ private static byte[] readPemBody(byte[] privateKeyBytes) throws IOException {
   private PrivateKey getPrivateKey() {
     Path keyFile = Path.of(conf.readMandatory(OAuth2Feature.Property.JWT_KEY_FILE));
     if (!keyFile.isAbsolute()) {
-      keyFile = ch.ivyteam.ivy.config.IFileAccess.instance().getConfigFile(keyFile.toString());
+      keyFile = ch.ivyteam.ivy.config.IFileAccess.instance().config().resolve(keyFile.toString());
```

Comment:
IFileAccess has changed:
this was the main reason which forced me to introduce a 'master' that run towards 11.2 only.
while the branch /release/10.0 remains to serve the actual LTS.